rules/integrations/aws_bedrock/aws_bedrock_high_confidence_blocks_detected.toml

```diff
@@ -0,0 +1,48 @@
+[metadata]
+creation_date = "2024/05/05"
+maturity = "production"
+updated_date = "2024/05/05"
+min_stack_comments = "ES|QL rule type is still in technical preview as of 8.13, however this rule was tested successfully; integration in tech preview"
+min_stack_version = "8.13.0"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies unusual patterns of high-confidence 'BLOCKED' actions by the model, which may indicate attempts to abuse the system.
+"""
+false_positives = ["New model deployments.", "Testing pdates to compliance policies."]
+from = "now-1d"
```

Comment:
Notice the other bedrock rules have `from = "now-60m"`

I can bump this down if it makes sense.

Reply:
Im not sure if there is a trade-off. We do this for most CSP rules, but it is now-60m for them.